examples/resnet-concurrent.cpp

```diff
@@ -196,9 +196,13 @@ int main(int argc, char **argv) {
                                   ImageChannelOrder::BGR, ImageLayout::NCHW,
                                   /* useImagenetNormalization */ true);
 
+    // The model expects a batch, but since we only want one item, the layout is
+    // the same and we can create an unowed tensor.
+    Tensor batch(image.getUnsafePtr(), inputType);
```

Comment:
I'd prefer `Tensor batch = image.getUnowned(inputType->dims());` -- I like to avoid calls to `getUnsafePtr()`, and `getUnowned()` includes asserts for the sizes to make sense.

Reply:
yeah thats much better, I was sure this would trip the asserts on dimensions since the new tensor is larger.